shared/build.gradle.kts

```diff
@@ -103,6 +103,7 @@ kotlin {
         }
 
         val iosMain by creating {
+            dependsOn(commonMain)
```

Comment:
Why `androidMain` not depends on `commonMain`? Or this syntax is optional? before everything works as expected.

![](https://resources.jetbrains.com/help/img/kotlin-multiplatform-dev/modules-structure.png)

Reply:
cc @ivan-magda

Reply:
OK, thank you for explanation